opm/simulators/aquifers/AquiferNumerical.hpp

```diff
@@ -0,0 +1,229 @@
+/*
+  Copyright (C) 2020 Equinor ASA
+  Copyright (C) 2020 SINTEF Digital
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_AQUIFERNUMERICAL_HEADER_INCLUDED
+#define OPM_AQUIFERNUMERICAL_HEADER_INCLUDED
+
+#include <opm/output/data/Aquifer.hpp>
+#include <opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/SingleNumericalAquifer.hpp>
+
+namespace Opm
+{
+template <typename TypeTag>
+class AquiferNumerical
```

Comment:
For numerical aquifers, they are part of the grid. So this class only handles the collection of the information for output.